impala/sqlalchemy.py

```diff
@@ -176,11 +176,28 @@ def get_columns(self, connection, table_name, schema=None, **kwargs):
         cursor.fetchall()
         column_info = []
         for col in schema:
-            column_info.append({
-                'name': col[0],
-                'type': _impala_type_to_sqlalchemy_type[col[1]],
-                'nullable': True,
-                'autoincrement': False})
+            try:
+                column_info.append({
+                    'name': col[0],
+                    'type': _impala_type_to_sqlalchemy_type[col[1]],
+                    'nullable': True,
+                    'autoincrement': False})
+            except KeyError as e:
+                if col[1] == 'VARCHAR':
```

Comment:
Can't you add these to _impala_type_to_sqlalchemy_type?

Impala already supports VARCHAR and DATE support is upcoming so this will be useful.